modules/local/attribute_in_gtf.nf

```diff
@@ -2,33 +2,26 @@
  * Check if attribute is available in column 9 of GTF file
  */
 process ATTRIBUTE_IN_GTF {
+    tag "$gtf"
+    
+    conda (params.enable_conda ? "conda-forge::sed=4.7" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://containers.biocontainers.pro/s3/SingImgsRepo/biocontainers/v1.2.0_cv1/biocontainers_v1.2.0_cv1.img"
+    } else {
+        container "biocontainers/biocontainers:v1.2.0_cv1"
+    }
 
     input:
-    val gtf
-    val attribute
+    path gtf
+    val  attribute
 
     output:
-    val attribute_in_gtf
+    env  (ATTRIBUTE_EXISTS), emit: exists

```

Comment:
I'm surprised that this doesn't need to be quoted..?

Reply:
Nope. It works just as fine :)